gitpod-io/gitpod
# Add beta label on usage

## Description

This will add a BETA label on the usage page with a link to send feedback.

We need a public feedback issue to link to. 💡 

Opened this after we briefly discussed this with @laushinka in a recent social call. 🍷 

## How to test
Go to usage and notice the new beta notice on the left sidebar.

<img width="269" alt="Screenshot 2022-08-25 at 11 59 52 PM" src="https://user-images.githubusercontent.com/120486/186767986-d866a540-3bee-42a5-a385-8b4347c662fd.png">

## Release Notes

```release-note
Add beta notice and label on usage
```

## Werft options:

- [x] /werft with-preview with-payment
